models/delta.coffee

```diff
@@ -90,15 +92,19 @@ class DeltaStream extends EventEmitter
   open: () ->
     @close()
     path = "/delta/streaming"
+    excludeTypes = @params.excludeTypes ? []
+    includeTypes = @params.includeTypes ? []
+
     queryObj =
       cursor: @cursor
-    queryObj.exclude_types = @excludeTypes.join(',') if @excludeTypes?.length > 0
-    queryStr = querystring.stringify(queryObj)
-    path += '?' + queryStr
+    queryObj.exclude_types = excludeTypes.join(',') if excludeTypes.length > 0
```

Comment:
Yeah, I think that was intentional, I don't want to add they param unless there is something present